jenkins/ci.suse.de/cloud-mkphyscloud-qa-scenario-3.yaml

```diff
@@ -179,6 +179,66 @@
           timeout --signal=ALRM 150m bash -x -c ". scripts/qa_crowbarsetup.sh ; onadmin_runlist $commands;"
           ' || ret=$?
 
+          #Test nova evacuate
+          OUTPUT=$(ssh -T controller1 <<'EOF' 
+          source /root/.openrc
+          # Get hypervisor name
+          hypervisor_host=$(nova hypervisor-list | grep enabled | grep 1 |cut -d '|' -f 3 | cut -d '.' -f 1 |cut -d ' ' -f 2)
+          echo $hypervisor_host
+          export hypervisor_host=$hypervisor_host
+          # Crate instance on specific hypervisor image always exist after tempest run (cirros-0.3.4-x86_64-tempest-machine)
+          nova boot --image cirros-0.3.4-x86_64-tempest-machine  --flavor  tempest-stuff  --availability-zone nova:$hypervisor_host NovaEvacuate
+          #Create floating ip assign to instance
+          floating_ip_id=$(neutron net-list | grep floating | cut -d ' ' -f 2)
+          #Create new floating ip address
+          floating_ip=$(neutron floatingip-create $floating_ip_id | grep floating_ip_address | cut -d " " -f 4)
+          #Update security group for icmp 
+          nova secgroup-add-rule default icmp -1 -1 0.0.0.0/0
+          #Assign public ip to instance
+          nova add-floating-ip NovaEvacuate  $floating_ip
+          #Check instance is ACTIVE and Running
+          ping -c 5 $floating_ip >/dev/null 2>&1
+          status=$(nova show  NovaEvacuate | grep status | cut -d '|' -f 3 | grep ACTIVE)
+          echo $floating_ip
+          if [ $status = "ACTIVE" ] && [ $? = 0 ] ; then
+          echo "Instance launched successfully"
+          else
+          echo  "Instance luanch failed"
+          fi
+          EOF
+          )
+          hypervisor_host=$(echo $OUTPUT | cut -d ' ' -f 1)
```

Comment:
Isn't this going to catch output from multiple lines of `$OUTPUT`?  What about if the `nova boot` fails, or any of the other commands here fail?  We need the whole of `$OUTPUT` visible in the build logs so that we can easily debug what went wrong.  So at least there should be an `echo -e "--- Starting $vm_name on controller1 ---\n$OUTPUT\n--- End of output ---"` or similar.